front/components/LP/reminder.tsx

```diff
@@ -8,14 +8,14 @@ export default class Reminder extends React.Component<ReminderProps, any> {
     return (
       <div className="reminder">
         <div className="heading">
-          Reminder? We send them.
+          Need a reminder? We'll send them
```

Comment:
For this, I feel 3 characters are better than having a `'ll`. My personal preference.